js/rpg_objects/Game_Interpreter.js

```diff
@@ -1730,3 +1739,53 @@ Game_Interpreter.prototype.command356 = function() {
 Game_Interpreter.prototype.pluginCommand = function(command, args) {
     // to be overridden by plugins
 };
+
+Game_Interpreter.requestImages = function(list){
+    list.forEach(function(command){
+        var params = command.parameters;
+        switch(command.code){
+            case 101:
```

Comment:
`Game_Interpreter` tends to have detailed comments because it is hard to see what the code number represents. Could you add comments like below?
```
case 101: // Show Text
```